hail/src/main/scala/is/hail/io/index/IndexWriter.scala

```diff
@@ -370,6 +370,8 @@ class StagedIndexWriter(branchingFactor: Int, keyType: PType, annotationType: PT
   }
   def close(cb: EmitCodeBuilder): Unit = {
     val off = flush.invokeCode[Long](cb)
+    cb += ob.flush()
+    cb += ob.close()
```

Comment:
I think we lost the removal of the other flushes in writeLeafNode and writeInternalNode.

Reply:
🤦 

no wonder it worked...